frontend/src/app/task-area/task-area.component.html

```diff
@@ -1,22 +1,41 @@
-<div style="height: calc(100vh - 200px)" class="wrapper work-area-pos">
+<div class="wrapper">
   <div class="working-area">
-    <form class="example-form">
-      <mat-form-field class="example-full-width" appearance="fill">
-        <mat-label>Print text of you task</mat-label>
-        <input matInput  />
-      </mat-form-field>
-      <button mat-button color="primary">Add new task</button>
-    </form>
+    <div class="side-bar">
+      <button class="btn active-tab">All</button>
+      <button class="btn">Groceries</button>
+      <button class="btn">College</button>
+      <button class="btn">Payments</button>
+    </div>
+    <div class="task-area">
```

Comment:
Task area should be separate component too.

Reply:
done